docs/settingup.md

```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
 }
 ```
 
+</details><details>
+<summary>subscribeobjects</summary>
+
+## Subscribeobjects action
+
+Subscribe to any object in the currently active app.
+
+### Settings
+
+* `clear`: Clear any previously subscribed objects from subscription list.
```

Comment:
Suggestion:
"Remove any previously subscribed objects from the subscription list."